app/main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+from __future__ import annotations
```

Comment:
remove redundant import

Reply:
31  def bite(self, animal: Herbivore | Carnivore) -> int | list:
For this annotation I've used from __future__ import annotations